frontend/src/Components/Orders/Row.js

```diff
@@ -0,0 +1,95 @@
+import { useState } from "react"
+import { SaveButton } from '../Common/SaveButton'
+import { Cancel } from '../Common/CancelButton'
+import { EditButton } from '../Common/EditButton'
+import { Edit } from '../Common/Edit'
+import axios from "axios"
+
+export function Row(props) {
+    const [isEditMode, setEditMode] = useState(false)
+    const [updatedOrder, setUpdatedOrder] = useState(props.order)
+
+    function onCancelClick() {
+        setEditMode(false)
+    }
+
+    function onClick() {
+        setEditMode(!isEditMode)
+        props.onClick()
+    }
+
+    function save(order) {
+        axios
+            .put('/orders/' + order.id,
```

Comment:
In loc de string concatenation as fi folosit in mod exclusiv template strings. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
Done.